tests/MyGame/Example/Any.cs

```diff
@@ -41,6 +41,31 @@ public class AnyUnion {
   }
 }
 
+
+
+static public class AnyVerify
+{
+  static public bool Verify(Google.FlatBuffers.Verifier verifier, byte typeId, uint tablePos)
+  {
+    bool result = false;
+    switch((Any)typeId)
+    {
+      case Any.Monster:
+        result = MyGame.Example.MonsterVerify.Verify(verifier, tablePos);
+        break;
+      case Any.TestSimpleTableWithEnum:
+        result = MyGame.Example.TestSimpleTableWithEnumVerify.Verify(verifier, tablePos);
+        break;
+      case Any.MyGame_Example2_Monster:
+        result = MyGame.Example2.MonsterVerify.Verify(verifier, tablePos);
+        break;
+      default: result = true;
```

Comment:
should the default be true?

Reply:
I think true is ok here: In the flatbuffers union is an type we don't know in our generated access code. So it could be that this new type was added later (in a compatible way). So we can access the flatbuffer because we will never access this field.